control-base/src/main/java/fi/nls/oskari/control/data/MapLayerGroupsHandler.java

```diff
@@ -70,12 +83,12 @@ public void handleGet(ActionParameters params) throws ActionException {
      */
     public void handlePut(ActionParameters params) throws ActionException {
         checkForAdminPermission(params);
-        final MaplayerGroup theme = new MaplayerGroup();
-        populateFromRequest(params, theme);
-        final int id = oskariMapLayerGroupService.insert(theme);
+        final MaplayerGroup maplayerGroup = new MaplayerGroup();
+        populateFromRequest(params, maplayerGroup);
```

Comment:
Instead of passing the `maplayerGroup` as a parameter to `populateFromRequest` we could create one inside the function and then return it.